backend/v1/components/aquariums/wrappers.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import HTTPException
+
+
+def validate_aquarium(func):
+    def wrapper(*args, **kwargs):
+        aquarium = args[0]
+        if None in aquarium.dict().values():
+            raise HTTPException(status_code=400, detail="Missing aquarium data")
+
+        if aquarium.name is None:
+            raise HTTPException(status_code=400, detail="Missing aquarium name")
+
+        if aquarium.temperature < 0 or aquarium.temperature > 100:
+            raise HTTPException(status_code=400, detail="Temperature must be between 0 and 100 degrees Celsius")
+
+        if aquarium.width < 0:
```

Comment:
Aquarium dimensions have to be larger than 0, I suggest adding a minimal width, height and length (sth like 5cm)

Reply:
todo in next commit